gerritlab/utils.py

```diff
@@ -67,7 +67,7 @@ def get_change_id(msg, silent=False):
 
 
 def get_remote_branch_name(local_branch, change_id):
```

Comment:
nit: might be worth renaming variables here since it's no longer looking at `local_branch`

I have the same thought about `gerritlab/merge_request.py` use of `_local_branch` (which is only used by the tests in `unit_tests/merge_request_test.py` afaict)